docs/source/develop/api/commands.rst

```diff
@@ -236,7 +236,7 @@ Validation
 ^^^^^^^^^^
 
 1. Caller is a valid Iroha account ID
-2. The transaction creator has a role with either CanCallEngine permission
+2. The transaction creator has a role with can_call_engine permission
```

Comment:
maybe the logically missing part was supposed to address `can_call_engine_on_my_behalf`, which we should add here